definitions/npm/next_v2.x.x/flow_v0.104.x-/test_next_v2.x.x.js

```diff
@@ -88,12 +88,12 @@ const p: string = Router.pathname;
 const q: { ... } = Router.query;
 
 export default class TestDocument extends Document {
-  static async getInitialProps (ctx: Context) {
+  static async getInitialProps (ctx: Context): Promise<any> {
     const props = await Document.getInitialProps(ctx)
     return { ...props, customValue: 'hi there!' }
   }
 
-  render () {
+  render(): React$Node {
```

Comment:
Is that really required?   
Also should rather be `React$Element<'html'>`.

Reply:
According to [flow documentation for react](https://flow.org/en/docs/react/types/#toc-react-node), it seems that `render()` should return a Node.

I'm not sure if we should set the node as optional.

Reply:
And without specifying it, CI tests won't pass

Reply:
I think those docs samples are a bit too generic, but anyway, it doesn't make a big difference here I guess.